### PR TITLE
Weekday is dependent on database

### PR DESCRIPTION
**Pull request by MarkvanMents**
[TW-1455]

### CODE REVIEW

File: content/refguide/xpath-weekday-from-datetime.md [hunk 1]
```diff
@@ -6,11 +6,15 @@ tags: ["studio pro"]
 
 ## 1 Overview
 
-The `weekday-from-dateTime()` function extracts the day of the week (as a number) from a **Date and time** attribute so it can be used to compare to a value. Values range from 1 to 7 (1 = Sunday, 7 = Saturday).
+The `weekday-from-dateTime()` function extracts the day of the week (as a number) from a **Date and time** attribute so it can be used to compare to a value.
+
+{{% alert type="info" %}}
+Values range from 1 to 7, but the day of the week corresponding to `1` depends on which database you are using. This means that the value you get in local testing can be different from the value you get in your production environment, and we recommend that you ensure that your testing takes this into account.
```
Comment (basdebakker) [inline content/refguide/xpath-weekday-from-datetime.md:12]: It's currently even worse than this. When using SAP HANA, we return values of 0 to 6.

Comment (MarkvanMents) [reply content/refguide/xpath-weekday-from-datetime.md:12]: Oh - I thought you mentioned that the range was always adjusted.
I'll update this.
Going to make it very difficult to test code locally and then implement…

**Comment by MarkvanMents** [comment]
@basdebakker - does this look better?

**Review by tknippenberg — APPROVED** [review]
Looks good!

**Pull request merged**